doc/OnlineDocs/source/concepts/ruc_sced_cycle.rst

```diff
@@ -1,42 +1,69 @@
+will go into effect at midnight, one at 8:00 a.m., and one at 4:00 p.m. Each
+RUC plan covers the time period that starts when it goes into effect and ends
+just as the next RUC plan becomes active.
+
+A RUC plan is based on the current state of the system at the time the plan is 
+generated (particularly up- and down-times for dispatchable generators), and on
+forecasts for a number of upcoming time periods. The forecasts considered when
+forming a RUC plan must extend at least to the end of the RUC's planning period,
+but typically extend further into the future in order to avoid poor choices at
+the end of the plan ("end effects"). The amount of time to consider when
+generating a RUC plan is known as the RUC horizon. A commonly used RUC horizon
+is 48 hours.
+
+The simulation can be configured to generate RUC plans some number of hours before
+they take effect. This is done by specifying a time of day for one of the plans to
+be generated. The gap between the specified generation time and the next time a RUC
+plan is scheduled to take effect is called the RUC gap. Each RUC plan still covers
+the expected time period, from the time the plan takes effect until the next RUC plan
+takes effect, but its decisions will be based on what is known at the time the RUC 
+plan is generated.
 
-The RUC phase occurs one or more times per day.  Each time the RUC phase occurs, Prescient generates
-a unit commitment schedule that indicates which generators will be brought online or taken offline
-within the RUC's time horizon.  The RUC schedule may begin immediately, or it may begin a number of
-hours after the RUC is generated.
+The SCED Cycle
+--------------
 
-The SCED Phase
-----------------
-More detailed description of the SCED, including a high level description of the optimization problem
-being solved, and possibly a conversational description of some things that can be tweaked (such as
-how often a SCED runs).
+The SCED process selects dispatch levels for all active dispatchable generators 
+in the current simulation time period. Dispatch levels are determined using a process
+that is very similar to that used to build a RUC plan. The current state of the
+system, together with forecasts for a number of future time periods, are examined to
+select dispatch levels that satisfy current loads and forecasted future loads at the
+lowest possible cost.
 
-.. _future-times-in-sceds:
+The SCED cycle is more frequent than the RUC cycle, with new dispatch levels selected 
+at least once an hour. The SCED honors unit commitment decisions made in the RUC plan;
+whether each generator is active or inactive is dictatated by the RUC schedule 
+currently in effect.
 
-Future Values in SCEDs
-``````````````````````
-.. warning::
-   Coming soon.
\ No newline at end of file
+Costs are also determined with each SCED, based on dispatchable generation selected by 
+the SCED process, as well as current actual demands and non-dispatchable generation levels.

```

Comment:
We should also say something about the commitment and startup costs, which are determined by the RUC, but the final calculation is pulled from the SCED.